include/repository/plugin_repository.js

```diff
@@ -221,6 +224,25 @@ module.exports = function PluginRepositoryModule(pb) {
         dao.q(PLUGIN_COLL, cb);
     };
 
+    /**
+     * Load a list of plugin ids from the current file system, except pencilblue as that's not a real plugin
+     * @returns {Promise<string[]>}
+     */
+    PluginRepository.loadAllPluginIds = async function () {
+        const readDirAsync = Promise.promisify(fs.readdir.bind(fs));
+        const statAsync = Promise.promisify(fs.stat.bind(fs));
+        const pluginsDir = pb.PluginService.PLUGINS_DIR;
+
+        let directoryContents = await readDirAsync(pluginsDir);
+        let fileStats = await Promise.props(
+            directoryContents.reduce((agg, item) => Object.assign(agg, {
+                [item]: statAsync(path.join(pluginsDir, item))
+            }), {}));
+
+        return directoryContents
+            .filter(id => id !== 'pencilblue' && fileStats[id].isDirectory());
```

Comment:
this should be comparing id against `pb.config.plugins.default` rather than `'pencilblue'`